inbox/api/jsc_api.py

```diff
@@ -118,3 +120,32 @@ def auth_callback():
         })
 
         return make_response( (resp, 201, { 'Content-Type': 'application/json' }))
+
+@app.route('/retry_sync', methods=['POST'])
+def retry_sync():
+    g.parser.add_argument('namespace_public_id', required=True, type=valid_public_id, location='form')
```

Comment:
About "namespace_public_id", we're calling it just "account_id" everywhere else and I think we should keep this pattern or even more confusion will arrive.